docs/source/ide_basics.rst

```diff
@@ -0,0 +1,131 @@
+IDE File Basics
+===============
+
+What's an IDE file?
+-------------------
+
+An IDE file is a read-only hierarchical data format that stores recording information generated by an enDAQ sensor device. It contains both time-indexed data from several different kinds of sensors (like acceleration, pressure, temperature, etc.), as well as metadata about the recording device (like device serial number, model number, device name, etc.) and recording settings.
+
+
+.. _dataset_desc:
+
+Accessing an IDE file
+---------------------
+
+The top-level interface for an IDE file is the :class:`Dataset` object, through which one can access all of the above-listed information. When you open a file for reading, for example, this is the type of object that is returned.
+
+Opening an IDE File
+~~~~~~~~~~~~~~~~~~~
+
+You can open an IDE file like so::
+
+    filename = "C:\\your\\file\\path.IDE"
```

Comment:
Ditch DOS-style paths

Reply:
addressed in f34863e2684b9ee9eddd485954b5e896cdc7e481